include/myst/cond.h

```diff
@@ -27,13 +28,20 @@ int myst_cond_timedwait(
     myst_cond_t* c,
     myst_mutex_t* mutex,
     const struct timespec* timeout);
+int myst_cond_timedwait_ops(
+    myst_cond_t* c,
+    myst_mutex_t* mutex,
+    const struct timespec* timeout,
+    uint32_t bitset);
 
 int myst_cond_signal(myst_cond_t* c);
+int myst_cond_signal_ops(myst_cond_t* c, uint32_t bitset);
```

Comment:
I think I would prefer changing the existing ``myst_cond_signal()`` prototype and adding a ``uint32* bitset`` parameter.

Reply:
`pipedev.c` and `eventfddev.c` uses the original function, meaning we would have to add bitset to that context, which doesn't necessarily need to know about it.

Reply:
You can pass null into those context.